### PR TITLE
fix: change shell command to generate masterkey

### PR DESCRIPTION
**Pull request by verdienansein**
## Description
I changed the command to generate masterKey on shell for Mia-Platform chart installation. The one before usually errors with: tr: Illegal byte sequence.

## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

File: YOUR_KEY_PATH [hunk 1]
```diff
@@ -0,0 +1 @@
+puN
```
Comment (fredmaggiowski) [inline YOUR_KEY_PATH:1]: why was this file checked in?

File: docs/runtime_suite/crud-service/30_encryption_configuration.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ try {
 
 #### How to generate a local master key using Shell Command
 ```shell
-cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 96 | head -n 1 > YOUR_KEY_PATH
+LC_CTYPE=C tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 96 | head -1
```
Comment (fredmaggiowski) [inline docs/runtime_suite/crud-service/30_encryption_configuration.md:138]: Why this locale set is necessary?

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/runtime_suite/crud-service/30_encryption_configuration.md (hunk 2 above)
Comment (verdienansein) [reply docs/runtime_suite/crud-service/30_encryption_configuration.md:138]: I found out that without using it I get an error on OSx:

```
tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 96 | head -1
tr: Illegal byte sequence
```

```
LC_CTYPE=C tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 96 | head -1
GNtv2tuSW78LhUZ66U5UVHX7FbEq3eTZ6UVtRxGRgvHDrOTyCWpDKVYeXIMBJvZ6VoLQLjIJjmpFWHCtmcojl583squs3C3I
```

Seems like a problem with the locale default on MacOSx as explained here: https://stackoverflow.com/questions/19242275/re-error-illegal-byte-sequence-on-mac-os-x

File: YOUR_KEY_PATH (hunk 1 above)
Comment (verdienansein) [reply YOUR_KEY_PATH]: My bad..

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Pull request closed without merging**